modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectField/Tabs/Advanced/AdvancedTab.tsx

```diff
@@ -38,20 +36,18 @@ export function AdvancedTab({
 
 	return (
 		<>
-			{Liferay.FeatureFlags['LPS-170122'] && isDefaultStorageType && (
-				<Card
+			<Card
```

Comment:
@murilostodolni You're still checking the value of `isDefaultStorageType` inside `EditObjectField`. Shouldn't you keep checking its value here also?